tests/Fixtures/Kernel.php

```diff
@@ -34,14 +38,18 @@ class Kernel extends BaseKernel
 
     private string $ormResetMode = ORMDatabaseResetter::RESET_MODE_SCHEMA;
 
+    private array $factoriesRegistered = [];
```

Comment:
I had a case where sometimes we want the factory registered in container, and sometimes not. So now, we can chose in the test which factory we need to be registered.